digitalutsc/islandora_web_annotations
# create required objects on install

## What does this PR do?
This PR addresses the issue of creating the required content models when installing the annotation module.  It resolves this issue: https://github.com/digitalutsc/islandora_web_annotations/issues/12

In addition, this PR cleans some initial test code.

## How can I test it?
* Completely uninstall the web annotation module
```
drush dis islandora_web_annotations && drush pm-uninstall islandora_web_annotations
```
Please note that you have to fully "uninstall" for the hooks to be fired as above.

* Delete the islandora:WADMCModel and islandora:WADMContainerCModel content model objects from Fedora
* Get this PR
* Re install the module
* The objects should be created in fedora.  A message saying the above two objects were created should be displayed in the Drupal UI.
